components/nordigen/sources/new-transaction/new-transaction.mjs

```diff
@@ -0,0 +1,152 @@
+import nordigen from "../../nordigen.app.mjs";
+import axios from "axios";
+
+// TODO: Add description docs
```

Comment:
@vellames-turing The description doc is TODO. Should I also add a package.json in the `nordigen` folder ?

Reply:
> > Hey @LucBerge. Great job! It's approved. I was not able to test it because even with your key, I am not able to complete the agreement part. Looks like it needs some personal information. But no need to share with me. Is it working as you expect right now?
> 
> Hey, great ! But I would like to try it in real condition first. Let's say 2 weeks to make sure an event is triggered for each transaction. I'll keep you in touch on this pull request.

I don't think you really need to wait for this. If your concern is only about the automatic trigger, it will work :)